### DISCUSSION TITLE
ErrorException Undefined index: s VENDORPATH\mpdf\mpdf\src\Tag\Br.php at line 60

### DISCUSSION THREAD

**Discussion by JazMagno** [mpdf/mpdf#1833] [Q&A]
I have checked that the html code is well formed. I get this error if I add a simple table with tbody and thead, if I leave the table without body or header it generates the pdf fine.

![image](https://user-images.githubusercontent.com/18746245/223647329-dee13e3c-7f01-4733-9e8b-9324d1db9a85.png)

**Comment by finwe** [comment]
share. **_your_**. code.

seriously, how do you think a screenshot of the library, that is available and anyone can look in the codebase based on the file and line of the notice helps in this case?

**Comment by JazMagno** [comment]
I found the solution. The problem was the content, the tables contained \r\n, removing them generated without problems.

Thank you.

**Reply by finwe** [reply]
Well If we had an example code, we could fix the library for other users.

**Reply by JazMagno** [reply]
This table causes problems:

```
<table>\r\n
<tbody>\r\n
<tr>\r\n
<td>Hola mundo</td>\r\n
</tr>\r\n
</tbody>\r\n
</table>
```

**Reply by finwe** [reply]
Not able to reproduce with this code example (when used in "" string in WriteHTML method)

**Reply by JazMagno** [reply]
Maybe this:

`<table style=\"border-collapse: collapse; width: 212.4pt; border-spacing: 0px; border: 2px solid rgb(0, 0, 0); margin-left: auto; margin-right: auto;\" border=\"1\" width=\"283\" cellspacing=\"0\" cellpadding=\"0\">\r\n<tbody>\r\n<tr style=\"mso-yfti-irow: 0; mso-yfti-firstrow: yes; height: 14.4pt;\">\r\n<td style=\"width: 212.4pt; border: 2px solid rgb(0, 0, 0); background: rgb(217, 217, 217); padding: 0cm 3.5pt; height: 14.4pt;\" colspan=\"4\" nowrap=\"nowrap\" width=\"283\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">CLC&acute;S</span></span></span></p>\r\n</td>\r\n</tr>\r\n<tr style=\"mso-yfti-irow: 1; height: 20.4pt;\">\r\n<td style=\"width: 49.4pt; border-width: 2px; border-style: none solid solid; border-color: rgb(0, 0, 0); border-image: initial; background: rgb(217, 217, 217); padding: 0cm 3.5pt; height: 20.4pt;\" width=\"66\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">Estimaci&oacute;n n&uacute;mero </span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 42.5pt; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); background: rgb(217, 217, 217); padding: 0cm 3.5pt; height: 20.4pt;\" nowrap=\"nowrap\" width=\"57\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">Folio</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 2cm; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); background: rgb(217, 217, 217); padding: 0cm 3.5pt; height: 20.4pt;\" nowrap=\"nowrap\" width=\"76\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">Fecha </span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 63.8pt; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); background: rgb(217, 217, 217); padding: 0cm 3.5pt; height: 20.4pt;\" nowrap=\"nowrap\" width=\"85\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">Monto </span></span></span></span></p>\r\n</td>\r\n</tr>\r\n<tr style=\"mso-yfti-irow: 2; height: 14.4pt;\">\r\n<td style=\"width: 49.4pt; border-width: 2px; border-style: none solid solid; border-color: rgb(0, 0, 0); border-image: initial; padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"66\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">1</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 42.5pt; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"57\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">14447</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 2cm; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"76\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">28-ago-20</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 63.8pt; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"85\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: right; line-height: normal;\" align=\"right\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span lang=\"ES-TRAD\" style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-ansi-language: ES-TRAD; mso-fareast-language: ES-MX;\">$822,490.87</span></span></span></span></p>\r\n</td>\r\n</tr>\r\n<tr style=\"mso-yfti-irow: 3; height: 14.4pt;\">\r\n<td style=\"width: 49.4pt; border-width: 2px; border-style: none solid solid; border-color: rgb(0, 0, 0); border-image: initial; padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"66\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">2</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 42.5pt; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"57\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">15684</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 2cm; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"76\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">24-sep-20</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 63.8pt; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"85\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: right; line-height: normal;\" align=\"right\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span lang=\"ES-TRAD\" style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-ansi-language: ES-TRAD; mso-fareast-language: ES-MX;\">$1,086,034.63</span></span></span></span></p>\r\n</td>\r\n</tr>\r\n<tr style=\"mso-yfti-irow: 4; height: 14.4pt;\">\r\n<td style=\"width: 49.4pt; border-width: 2px; border-style: none solid solid; border-color: rgb(0, 0, 0); border-image: initial; padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"66\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">3</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 42.5pt; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"57\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">16464</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 2cm; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"76\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">01-oct-20</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 63.8pt; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"85\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: right; line-height: normal;\" align=\"right\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span lang=\"ES-TRAD\" style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-ansi-language: ES-TRAD; mso-fareast-language: ES-MX;\">$1,434,002.58</span></span></span></span></p>\r\n</td>\r\n</tr>\r\n<tr style=\"mso-yfti-irow: 5; height: 14.4pt;\">\r\n<td style=\"width: 49.4pt; border-width: 2px; border-style: none solid solid; border-color: rgb(0, 0, 0); border-image: initial; padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"66\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">4</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 42.5pt; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"57\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">21800</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 2cm; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"76\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">28-dic-20</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 63.8pt; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"85\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: right; line-height: normal;\" align=\"right\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span lang=\"ES-TRAD\" style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-ansi-language: ES-TRAD; mso-fareast-language: ES-MX;\">$1,620,718.58</span></span></span></span></p>\r\n</td>\r\n</tr>\r\n<tr style=\"mso-yfti-irow: 6; height: 14.4pt;\">\r\n<td style=\"width: 49.4pt; border-width: 2px; border-style: none solid solid; border-color: rgb(0, 0, 0); border-image: initial; padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"66\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">5</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 42.5pt; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"57\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">2258</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 2cm; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"76\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">31-dic-20</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 63.8pt; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"85\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: right; line-height: normal;\" align=\"right\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span lang=\"ES-TRAD\" style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-ansi-language: ES-TRAD; mso-fareast-language: ES-MX;\">$1,660,908.55</span></span></span></span></p>\r\n</td>\r\n</tr>\r\n<tr style=\"mso-yfti-irow: 7; height: 14.4pt;\">\r\n<td style=\"width: 49.4pt; border-width: 2px; border-style: none solid solid; border-color: rgb(0, 0, 0); border-image: initial; padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"66\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">6</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 42.5pt; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"57\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">22559</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 2cm; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"76\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">28-dic-20</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 63.8pt; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"85\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: right; line-height: normal;\" align=\"right\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span lang=\"ES-TRAD\" style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-ansi-language: ES-TRAD; mso-fareast-language: ES-MX;\">$1,434,407.21</span></span></span></span></p>\r\n</td>\r\n</tr>\r\n<tr style=\"mso-yfti-irow: 8; mso-yfti-lastrow: yes; height: 14.4pt;\">\r\n<td style=\"width: 49.4pt; border-width: 2px; border-style: none solid solid; border-color: rgb(0, 0, 0); border-image: initial; padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"66\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">7</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 42.5pt; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"57\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">1985</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 2cm; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"76\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: center; line-height: normal;\" align=\"center\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-fareast-language: ES-MX;\">20-mar-21</span></span></span></span></p>\r\n</td>\r\n<td style=\"width: 63.8pt; border-width: 2px; border-style: none solid solid none; border-color: rgb(0, 0, 0); padding: 0cm 3.5pt; height: 14.4pt;\" nowrap=\"nowrap\" width=\"85\">\r\n<p class=\"MsoNormal\" style=\"margin-bottom: 0cm; text-align: right; line-height: normal;\" align=\"right\"><span style=\"mso-bookmark: OLE_LINK3;\"><span style=\"mso-bookmark: OLE_LINK2;\"><span style=\"mso-bookmark: OLE_LINK1;\"><span lang=\"ES-TRAD\" style=\"font-size: 8.0pt; mso-ascii-font-family: Calibri; mso-fareast-font-family: 'Times New Roman'; mso-hansi-font-family: Calibri; mso-bidi-font-family: Calibri; color: black; mso-ansi-language: ES-TRAD; mso-fareast-language: ES-MX;\">$1,288,431.37&nbsp;&nbsp;</span></span></span></span></p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>`

**Reply by finwe** [reply]
No, even this runs OK in the last mPDF version.

**Comment by LorenzKeel** [comment]
I my case I had this error because of nl2br function
